TestAssets/DesktopTestProjects/DesktopAppWhichCallsDotnet/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*"
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-rc2-*"
   },
```

Comment:
using this regex: `\"Microsoft\.DotNet\.Cli\.Utils\"\s*:\s*\"1\.0\.0-[^p]`

This Is the only one I'm finding that isn't right